pkg/manage/cmd_initial_data.go

```diff
@@ -0,0 +1,101 @@
+package manage
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/OpenSlides/openslides-manage-service/pkg/datastore"
+	"github.com/OpenSlides/openslides-manage-service/proto"
+	"github.com/spf13/cobra"
+)
+
+const initialDataHelp = `Creates initial data if there is an empty datastore
+
+This command also sets password of user 1 to the value in the docker secret "admin".
+
+It does nothing if the datastore is not empty.
+`
+
+// CmdInitialData creates given initial data if there is an empty datastore. It
+// also sets password of user 1 to the value in the docker secret "admin".
+// This does nothing if the datastore is not empty.
+func CmdInitialData(cfg *ClientConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "initial-data",
+		Short: "Creates initial data if there is an empty datastore.",
+		Long:  initialDataHelp,
+	}
+
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		ctx, cancel := context.WithTimeout(context.Background(), cfg.Timeout)
+		defer cancel()
+
+		service, close, err := Dial(ctx, cfg.Address)
+		if err != nil {
+			return fmt.Errorf("connecting to gRPC server: %w", err)
+		}
+		defer close()
+
+		// TODO: Use initial data from file given in an env variable.
+		d := `{"foo":"bar"}`
+		req := &proto.InitialDataRequest{
+			Data: d,
+		}
+
+		resp, err := service.InitialData(ctx, req)
+		if err != nil {
+			return fmt.Errorf("setting initial data: %w", err)
+		}
+
+		msg := "Datastore contains data. Initial data were NOT set."
+		if resp.Initialized {
+			msg = "Initial data were set successfully."
+		}
+		fmt.Println(msg)
+
+		return nil
+	}
+
+	return cmd
+}
+
+// Sets initial data in datastore.
+func (s *Server) InitialData(ctx context.Context, in *proto.InitialDataRequest) (*proto.InitialDataResponse, error) {
+	initialData, err := parseData(in.Data)
+	if err != nil {
+		return nil, fmt.Errorf("parsing initial data: %w", err)
+	}
+
+	const magicKey = "organisation/1/id"
+	var existingData bool
```

Comment:
change to int

Reply:
No I use an EXIST call to check if any organisation element with id 1 exists. Is this sufficient?